orc8r/gateway/c/common/async_grpc/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "async_grpc",
+    srcs = [
+        "GRPCReceiver.cpp",
+    ],
+    hdrs = [
+        "includes/GRPCReceiver.h",
```

Comment:
Yeah I think it was added to appease CMake, so if we do transition to Bazel we can act on that issue. (Will put it into the repo as a TODO so that I remember to do this.)